version 0.8.0 fails when running `chart add` command

When running the latest version (arm64 release), I get the following error when trying to add a chart:
```
kcl chart add -c kyverno -r https://kyverno.github.io/kyverno
panic: unable to redefine 't' shorthand in "add" flagset: it's already used for "target_revision" flag

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0x14000c38100, 0x14000c2fae0)
        github.com/spf13/pflag@v1.0.6/flag.go:881 +0x378
github.com/spf13/cobra.(*Command).mergePersistentFlags.(*FlagSet).AddFlagSet.func2(0x14000c2fae0)
        github.com/spf13/pflag@v1.0.6/flag.go:894 +0x44
github.com/spf13/pflag.(*FlagSet).VisitAll(0x14000c36908?, 0x14000b9bbd8)
        github.com/spf13/pflag@v1.0.6/flag.go:297 +0xdc
github.com/spf13/pflag.(*FlagSet).AddFlagSet(...)
        github.com/spf13/pflag@v1.0.6/flag.go:892
github.com/spf13/cobra.(*Command).mergePersistentFlags(0x14000c36908)
        github.com/spf13/cobra@v1.9.0/command.go:1898 +0x84
github.com/spf13/cobra.stripFlags({0x14000c3a0c0, 0x4, 0x4}, 0x14000c36908)
        github.com/spf13/cobra@v1.9.0/command.go:677 +0x3c
github.com/spf13/cobra.(*Command).Find.func1(0x14000c36908, {0x14000c3a0c0, 0x4, 0x4})
        github.com/spf13/cobra@v1.9.0/command.go:760 +0x48
github.com/spf13/cobra.(*Command).Find.func1(0x14000c36308, {0x14000b7fae0, 0x5, 0x5})
        github.com/spf13/cobra@v1.9.0/command.go:768 +0xa8
github.com/spf13/cobra.(*Command).Find.func1(0x14000af5508, {0x1400015e2b0, 0x6, 0x6})
        github.com/spf13/cobra@v1.9.0/command.go:768 +0xa8
github.com/spf13/cobra.(*Command).Find(0x300000001?, {0x1400015e2b0?, 0x109a25d58?, 0x18?})
        github.com/spf13/cobra@v1.9.0/command.go:773 +0x48
main.bootstrapCmdPlugin(0x1055c47b0?, {0x1067f6a80, 0x14000c01b30})
        github.com/MacroPower/kclipper/cmd/kclipper/main.go:99 +0x164
main.main()
        github.com/MacroPower/kclipper/cmd/kclipper/main.go:48 +0xbc
```

Running the same command with version `0.7.0` works fine.

--- a/internal/cli/chart.go
+++ b/internal/cli/chart.go
@@ -53,7 +53,7 @@ func NewChartCmd() *cobra.Command {
 		panic(err)
 	}
 
-	cmd.PersistentFlags().DurationP("timeout", "t", 5*time.Minute, "Timeout for the command")
+	cmd.PersistentFlags().Duration("timeout", 5*time.Minute, "Timeout for the command")
 
 	cmd.AddCommand(NewChartInitCmd())
 	cmd.AddCommand(NewChartAddCmd())

--- a/internal/cli/chart_test.go
+++ b/internal/cli/chart_test.go
@@ -1,0 +1,95 @@
+package cli_test
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/MacroPower/kclipper/internal/cli"
+)
+
+func TestChartCmd(t *testing.T) {
+	t.Parallel()
+
+	err := os.RemoveAll(filepath.Join(testDataDir, "got/chart_cmd"))
+	require.NoError(t, err)
+
+	tc := cli.NewRootCmd("test_chart", "", "")
+	out := bytes.NewBufferString("")
+
+	tc.SetArgs([]string{
+		"chart", "init",
+		"--path", filepath.Join(testDataDir, "got/chart_cmd"),
+	})
+	tc.SetOut(out)
+	tc.SetErr(out)
+	err = tc.Execute()
+	require.NoError(t, err)
+	require.Empty(t, out.String())
+	out.Reset()
+
+	// Replace `modules/helm` in kcl.mod with the correct path.
+	modFile := filepath.Join(testDataDir, "got/chart_cmd/kcl.mod")
+	modData, err := os.ReadFile(modFile)
+	require.NoError(t, err)
+	modData = bytes.ReplaceAll(modData,
+		[]byte(`helm = { path = "../modules/helm" }`),
+		[]byte(`helm = { path = "../../../../../modules/helm" }`),
+	)
+	err = os.WriteFile(modFile, modData, 0o600)
+	require.NoError(t, err)
+
+	tc.SetArgs([]string{
+		"chart", "add",
+		"--path", filepath.Join(testDataDir, "got/chart_cmd"),
+		"--chart=podinfo",
+		"--target_revision=6.7.0",
+		"--repo_url=https://stefanprodan.github.io/podinfo",
+	})
+	tc.SetOut(out)
+	tc.SetErr(out)
+	err = tc.Execute()
+	require.NoError(t, err)
+	require.Empty(t, out.String())
+	out.Reset()
+
+	tc.SetArgs([]string{
+		"chart", "set",
+		"--path", filepath.Join(testDataDir, "got/chart_cmd"),
+		"--chart=podinfo",
+		"-O", "targetRevision=6.7.1",
+	})
+	tc.SetOut(out)
+	tc.SetErr(out)
+	err = tc.Execute()
+	require.NoError(t, err)
+	require.Empty(t, out.String())
+	out.Reset()
+
+	tc.SetArgs([]string{
+		"chart", "repo", "add",
+		"--path", filepath.Join(testDataDir, "got/chart_cmd"),
+		"--name=stefanprodan",
+		"--url=https://stefanprodan.github.io/podinfo",
+	})
+	tc.SetOut(out)
+	tc.SetErr(out)
+	err = tc.Execute()
+	require.NoError(t, err)
+	require.Empty(t, out.String())
+	out.Reset()
+
+	tc.SetArgs([]string{
+		"chart", "update",
+		"--path", filepath.Join(testDataDir, "got/chart_cmd"),
+	})
+	tc.SetOut(out)
+	tc.SetErr(out)
+	err = tc.Execute()
+	require.NoError(t, err)
+	require.Empty(t, out.String())
+	out.Reset()
+}

--- a/internal/cli/root.go
+++ b/internal/cli/root.go
@@ -1,12 +1,20 @@
 package cli
 
 import (
+	"sync"
+
 	"github.com/spf13/cobra"
 
 	kclcmd "kcl-lang.io/cli/cmd/kcl/commands"
 )
 
+// Global lock for KCL command creation.
+var mu sync.Mutex
+
 func NewRootCmd(name, shortDesc, longDesc string) *cobra.Command {
+	mu.Lock()
+	defer mu.Unlock()
+
 	cmd := &cobra.Command{
 		Use:           name,
 		Short:         shortDesc,

--- a/internal/cli/root_test.go
+++ b/internal/cli/root_test.go
@@ -20,13 +20,16 @@ func init() {
 	testDataDir = filepath.Join(dir, "testdata")
 }
 
-func TestRun(t *testing.T) {
+func TestRunCmd(t *testing.T) {
 	t.Parallel()
 
-	tc := cli.NewRootCmd("test", "", "")
+	err := os.RemoveAll(filepath.Join(testDataDir, "got/run_cmd"))
+	require.NoError(t, err)
+
+	tc := cli.NewRootCmd("test_run", "", "")
 	out := bytes.NewBufferString("")
-	outFile := filepath.Join(testDataDir, "got/simple.json")
-	err := os.MkdirAll(filepath.Dir(outFile), 0o750)
+	outFile := filepath.Join(testDataDir, "got/run_cmd/simple.json")
+	err = os.MkdirAll(filepath.Dir(outFile), 0o750)
 	require.NoError(t, err)
 
 	tc.SetArgs([]string{"run", filepath.Join(testDataDir, "simple.k"), "--format=json", "--output", outFile})
@@ -44,9 +47,7 @@ func TestRun(t *testing.T) {
 
 func BenchmarkRun(b *testing.B) {
 	for range b.N {
-		tc := cli.NewRootCmd("test", "", "")
-
-		// plugin.RegisterPlugin(helm.HelmPlugin)
+		tc := cli.NewRootCmd("bench_run", "", "")
 
 		out := bytes.NewBufferString("")
 

